scribus/plugins/import/pdf/slaoutput.h

```diff
@@ -153,6 +153,111 @@ class AnoOutputDev : public OutputDev
 	QStringList *m_importedColors;
 };
 
+/* PDF TextBox Framework */
+/*
+* Holds all the details for each glyph in the text imported from the pdf file.
+*
+*/
+struct PdfGlyph 
+{
+	double dx;  // X advance value
+	double dy;  // Y advance value
+	double rise;    // Text rise parameter
+	QChar code;   // UTF-16 coded character
+};
+
+
+class TextRegionLine
+{
+public:
+	qreal maxHeight = {};
+	//we can probably use maxHeight for this.	
+	qreal width = {};
+	int glyphIndex = {};
+	QPointF baseOrigin = QPointF({}, {});
+	std::vector<TextRegionLine> segments = std::vector<TextRegionLine>();
+
+};
+
+class TextRegion
+{
+public:
+	enum class FrameworkLineTests 

```

Comment:
needs a better name... but i don't 100% get what the common meaning of the values is.

`TextRegion::LineType`?

Reply:
done